Documentation/Available-Snapshot-Strategies.md

```diff
@@ -821,9 +821,6 @@ assertSnapshot(
   matching: vc,
   as: .image(traits: .init(horizontalSizeClass: .compact))
 )
-
-// Match reference as-is.
```

Comment:
the same thing appears already in 
https://github.com/pointfreeco/swift-snapshot-testing/pull/745/files#diff-41c2a928fc8c7f668eae4d9c4a54f238c917a424b98c9e4deeb86c364cb7249cR804